com.unity.ml-agents/Runtime/Actuators/ActionSpec.cs

```diff
@@ -94,5 +94,35 @@ internal void CheckAllContinuousOrDiscrete()
                 );
             }
         }
+
+        public static ActionSpec Combine(ActionSpec[] specs)
```

Comment:
Is this used, or is it from earlier when we were only using a single actuator?

Reply:
from a single, it's also on the ActuatorComponent base class, so I was figuring I would need to use it for that as well.